packages/ethereum-ownership-pcd/test/EthereumOwnershipPCD.spec.ts

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { ArgumentTypeName } from "@pcd/pcd-types";
+import {
+  SemaphoreIdentityPCDPackage,
+  SemaphoreIdentityPCDTypeName,
+} from "@pcd/semaphore-identity-pcd";
+import { Identity } from "@semaphore-protocol/identity";
+import assert from "assert";
+import { ethers } from "ethers";
+import * as path from "path";
+import { EthereumOwnershipPCDPackage } from "../src/EthereumOwnershipPCD";
+
+const zkeyFilePath: string = path.join(__dirname, "../artifacts/16.zkey");
+const wasmFilePath: string = path.join(__dirname, "../artifacts/16.wasm");
+
+describe("Ethereum ownership PCD", function () {
+  this.timeout(30 * 1000);
```

Comment:
30 seconds is a very long time!

Reply:
yes, locally each of these tests pass v fast, on CI they take slightly longer than the default timeout (b/c the CI machines are tiny and slow), so I set 30s just to be safe